drivers/sensor/qdec_mcux/qdec_mcux.c

```diff
@@ -48,11 +54,15 @@ static int qdec_mcux_attr_set(const struct device *dev, enum sensor_channel ch,
 		return -ENOTSUP;
 	}
 
-	switch (attr) {
+	switch ((int) attr) {
```

Comment:
nit: prefer a cast to `enum sensor_attribute_qdec_mcux` instead (same for line 80)

Reply:
Fixed. Also fixed the BUILD_ASSERT check.